molgenis-data-mapper/src/main/java/org/molgenis/data/mapper/repository/impl/AttributeMappingRepositoryImpl.java

```diff
@@ -75,6 +77,23 @@ public AttributeMapping apply(Entity attributeMappingEntity)
 		});
 
 	}
+	
+	@Override
+	public List<AttributeMetaData> retrieveAttributeMetaDatasFromAlgorithm(String algorithm,
```

Comment:
Attributes specified in comments are shown in the results table as a separate column, but do not have any influence on the result value. I guess this can be considered confusing, or it works as you might expect